backend/imports/tasks/methods.js

```diff
@@ -64,8 +64,8 @@ export const checkTask = function ({ _id, checked }) {
  * @param _id {string}
  * @return {number} 1 if successfull, otherwise 0
  */
-export const removeTask = function ({ _id }) {
-  const userId = this.userId
-  checkUser(userId)
-  return TasksCollection.remove({ _id, userId })
-}
+export const removeTask = async function ({ _id }) {
+  const userId = this.userId;
+  checkUser(userId);
+  return await TasksCollection.remove({ _id, userId });
```

Comment:
shouldn't it be `await TasksCollection.removeAsync` ?

Reply:
Yes you are right